invoice/pdf_generator.py

```diff
@@ -90,17 +91,17 @@ def render_lines_left_right(x, y, lines, line_offset=0, line_height=16):
 
     # Totals
     render_lines_left_right(-(A4_WIDTH - x_left), table_y_end, [
-        ["Net Total: ", invoice.net_total_string],
-        ["Total: ", invoice.total_string]
+        [f'{gettext("Net Total")}: ', invoice.net_total_string],
```

Comment:
these should use formatting instead of simple concatenation

Reply:
sorry, what do you mean by that?

Reply:
you are concatenating the number and the translation of "Net Total ".
It should instead use some kind of formatting. I dont know the API, might be `gettext_format("Net Total", amount=invoice.net_total_string)` or something.

Reply:
Like this? Will changes others then.

Reply:
Formatting with one string does break the code as it expects the row to have two elements.

Reply:
oh yeah, i guess this special case needs to stay separate

Reply:
But maybe some other cases that combine text with numbers or text with text. Those should be formatted together so a translator can move the number freely in the text snippet and maybe even apply plural rules

Reply:
Got it. Will do

Reply:
I checked there are no sentence like structures in the pdf gen that require it. And we would end up with `gettext` formats within an f-string.
I prefer to leave it as such.